src/helpers/getNavLink.test.js

```diff
@@ -0,0 +1,17 @@
+import getNavLink from './getNavLink';
+
+describe('helpers/getNavLink', () => {
+  const inputArray = ['page', 'abc123', 'section'];
+
+  it('should return a shortened array if the same id is selected', () => {
```

Comment:
Same as above. It's easier to understand when we say what this shortened array _actually_ is, consider something like `current route` or `route parts`